lib/upload-apperian.js

```diff
@@ -190,28 +215,31 @@ var main = (function () {
 	function findAppId() {
 		doLog && console.log(LOG_TAG, '- getAppId');
 		var params = {
-			token: transactionToken
+			headers: {
+				"X-TOKEN": transactionToken
+			}
 		};
-		executeMethod(METHOD_GETLIST, params, function (_response) {
-			var type = '';
+		executeRequest(METHOD_GETLIST, params, function (_response) {
+			var type = [];
 			switch (appType) {
-			case 'ios':
-				type = 'IPA';
-				break;
-			case 'android':
-				type = 'APK';
-				break;
-			case 'microsoft':
-				type = 'APPX';
-				break;
-			default:
+				case 'ios':
+					type.push(1); // iOS
```

Comment:
I'd probably have these as constants and add a code comment where they are coming from